src/Select/Select.stories.tsx

```diff
@@ -15,6 +15,8 @@ storiesOf('Select', module).add(
         invalid={boolean('invalid', false)}
         invalidText={text('invalidText', 'A valid value is required')}
         invalidTextColor={text('invalidTextColor', '')}
+        borderColor={text('borderColor', 'red')}
```

Comment:
Let's default this to an empty string rather than 'red'.

Reply:
Fixed